src/System.Threading.Tasks.Analyzers/Setup/source.extension.vsixmanifest

```diff
@@ -13,9 +13,9 @@
     <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
   </Dependencies>
   <Assets>
-    <Asset Type="Microsoft.VisualStudio.Analyzer" Path="System.Threading.Tasks.Analyzers.dll" />
-    <Asset Type="Microsoft.VisualStudio.Analyzer" Path="System.Threading.Tasks.CSharp.Analyzers.dll" />
-    <Asset Type="Microsoft.VisualStudio.Analyzer" Path="System.Threading.Tasks.VisualBasic.Analyzers.dll" />
+    <Asset Type="Microsoft.VisualStudio.Analyzer" Path="System.Threading.Tasks.Analyzers.dll" IsOptionBased="true" />
```

Comment:
I think we probably need to first finalize the design here? Can we discuss this at the next CodeAnalysis design meeting?